hphp/test/frameworks/results/mockery.expect

```diff
@@ -727,8 +739,6 @@ Mockery_MockTest::testShouldAllowMockingProtectedMethodReturnsMockInstance
 NamedMockTest::itCreatesANamedMock
 .
 NamedMockTest::itCreatesPassesFurtherArgumentsJustLikeMock
-FATAL
-NamedMockTest::itShouldThrowIfAttemptingToRedefineNamedMock
```

Comment:
Was there intended to be a FATAL in a .expects? Note, I can add this back with a re-record - just not certain of whether they are intended to be retained.